aiven/client/cliarg.py

```diff
@@ -22,15 +23,15 @@ def get_json_config(path_or_string):
     return jsonlib.loads(path_or_string)
 
 
-def json_path_or_string(param_name):
-    def wrapper(fun):
+def json_path_or_string(param_name: str) -> Callable[[Callable[[CommandLineTool], Any]], Callable[[CommandLineTool], Any]]:
```

Comment:
I'd use a type variable instead of `Any` for the result here and for all internal functions.